ascend-sdk/src/main/java/ai/evolv/ascend/android/Allocations.java

```diff
@@ -49,4 +52,44 @@ JsonObject getGenomeFromAllocations() {
         genomeWrapped.add("genome", genome);
         return genomeWrapped;
     }
+
+    static JsonArray reconcileAllocations(JsonArray previousAllocations, @NotNull JsonArray currentAllocations) {
+        /*
```

Comment:
Can you use JavaDocs format in these?

https://google.github.io/styleguide/javaguide.html

Example:

/**
 * Returns an Image object that can then be painted on the screen. 
 * The url argument must specify an absolute {@link URL}. The name
 * argument is a specifier that is relative to the url argument. 
 * <p>
 * This method always returns immediately, whether or not the 
 * image exists. When this applet attempts to draw the image on
 * the screen, the data will be loaded. The graphics primitives 
 * that draw the image will incrementally paint on the screen. 
 *
 * @param  url  an absolute URL giving the base location of the image
 * @param  name the location of the image, relative to the url argument
 * @return      the image at the specified URL
 * @see         Image
 */
 public Image getImage(URL url, String name) {

Reply:
Ok, Im using proper javadoc in the reset of the project. Ill change this one though.